common/src/main/java/com/sequenceiq/cloudbreak/auth/altus/Crn.java

```diff
@@ -159,7 +159,8 @@ public static Partition safeFromString(String input) {
         FREEIPA("freeipa", NON_ADMIN_SERVICE),
         DATAHUB("datahub", NON_ADMIN_SERVICE),
         DATALAKE("datalake", NON_ADMIN_SERVICE),
-        DEX("dex", NON_ADMIN_SERVICE),
+        DEX("dex", NON_ADMIN_SERVICE), // TODO: remove this once all DEX enviroments migrate over to DE
```

Comment:
@arunmahadevan Checkstyle is failing with "Don't use trailing comments." You should follow the style we use for deprecating these. Refer to "SDX" above.

```
/**
  * @deprecated {@link #DEX} was replaced by {@link #DE} and is kept here for backwards compatibility.
  */
@Deprecated
DEX("dex", NON_ADMIN_SERVICE)
```

Reply:
Updated.